docs/02-get-started/01-quick-install.md

```diff
@@ -42,22 +42,12 @@ export CLUSTER_DOMAIN={YOUR_CLUSTER_DOMAIN}
 
 To manage Kyma via GUI, connect it to Kyma Dashboard. 
 
-1. To start the Dashboard, run:
+   To start the Dashboard, run:
 
     ```bash
     kyma dashboard
     ```
-    This command opens the Dashboard under [`http://localhost:3001/`](http://localhost:3001/).
-    
-2. Click the button to add your cluster to the Dashboard. 
-3. [Get your `kubeconfig` file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). Paste it into a text editor and replace the `0.0.0.0` part in the **cluster.server** value for k3d with `host.docker.internal`. <!-- //TODO: Remove the info about replacing the **cluster.server** value when this gets fixed. -->
-4. Upload it into the Dashboard as prompted.
-
-This takes you to your Kyma Dashboard.
-
-<!--
-//TODO: Finish when the following get fixed: Replace the Docker command in step 1 with `kyma dashboard --local` and the address in step 2 with `https://dashboard.kyma-project/`.
--->
+   This command takes you to your Kyma Dashboard under [`http://localhost:3001/`](http://localhost:3001/).
```

Comment:
```suggestion
This command takes you to your Kyma Dashboard under [`http://localhost:3001/`](http://localhost:3001/).
```

As this is not a list of steps anymore, we must fix the indentation (i.e., remove it).